cloud/passportauth/src/auth/PassportSetup.ts

```diff
@@ -0,0 +1,79 @@
+import * as express from 'express';
+import * as session from 'express-session';
+import {SessionOptions} from 'express-session';
+import * as passport from 'passport';
+import {Strategy} from 'passport-local';
+import {UserSec} from '../user/UserSec';
+
+export interface Setup {
+  init(app: express.Express, sessionOpts: SessionOptions): void;
+  setStrategy(): void;
+  setSerializeUser(): void;
+  setDeserializeUser(): void;
```

Comment:
These empty signatures are implementation details, can't enforce them through an interface.

These look like they would be better fitting as a set of options:
```typescript
serialize: (user: User, cb: (error: Error, id: string) => any) => any;
deserialize: (id: string, cb: (error: Error, user: User) => any) => any;
```